README.md

```diff
@@ -30,6 +30,7 @@ The yml file should be in the following format:
 name: Testing book
 identifier: com.ashfurrow.example
 resources: assets # Optional
+cover: cover.jpeg # Optional
```

Comment:
JPEG and PNG both work, I should add documentation that the image should be 4x3 (Apple's are 400x300, actually).